tools/cldr-code/src/test/java/org/unicode/cldr/unittest/TestBCP47.java

```diff
@@ -78,9 +78,6 @@ public static void main(String[] args) {
     }
 
     public void TestEnglishKeyTranslations() {
-        logKnownIssue(
-                "cldr7631",
-                "Using just warnings for now, until issues are resolved. Change WARNING/ERROR when removing this.");
```

Comment:
CLDR-7631 was resolved a while ago

Reply:
I don't know why my sweep didn't find this.  Is this function actually called?

Reply:
@srl295 Maybe because until a day or so ago the tests in this file were not being run (since they were not included in TestAl())?